packages/worker/src/chainSubscribers.ts

```diff
@@ -335,12 +335,22 @@ class ChainSubscribers {
                 const userAddress = parsedLog.args[1];
 
                 if (community) {
+                    // add subgraph beneficiary
+                    await database.models.subgraphUBIBeneficiary.create({
```

Comment:
what if the beneficiary is removed from one community and added to another one?

Reply:
the `userAddress` is not a unique key on this table. So, in this case will have 2 registries about the same user, but one with the state 0 and other state 1 - removed.

Reply:
ahh right, because you need to keep the old one. Thanks